README.md

```diff
@@ -985,6 +985,7 @@ Table of Contents
    * [cockroachlabs.com](https://www.cockroachlabs.com/free-tier/) — Free CockroachDB up to 5GB and 1vCPU.
    * [Macrometa](https://www.macrometa.com/) - a noSQL database, Pub/Sub, event processing, and serverless edge computing platform for building geo-distributed and real-time applications. Free dev account gives access to 10,000 Operations/Day & 200MB Storage.
    * [Planetscale](https://planetscale.com/) - PlanetScale is a MySQL compatible, serverless database platform powered by Vitess, 3 databases for free with 10GB storage, 100 Million rows read/mo per database, and 10 Million rows written/mo per database.
+   * [YugabyteDB](http://cloud.yugabyte.com) - YugabyteDB is a distributed SQL database compatible with PostgresSQL. The cloud free tier is a 2 vCPU, 4GB RAM, 10GB Disk.
```

Comment:
Probably https://www.yugabyte.com/cloud/ would be a better link, however I am not sure it fits really.  What does "30-day test sandbox" mean? data only kept that long? trial only that long? its not clear

Reply:
This is an old page from the beta, now redirecting to http://cloud.yugabyte.com 
The free tier has now no limit

Reply:
Unfortunately when I look at the page `cloud.yugabyte.com` there is zero information at all about the plans or offering only a login page, I cannot verify that this is a free offering.

Reply:
Ok, I understand. Conditions are there once signed-up. I'll update here when the landing page of  https://www.yugabyte.com/cloud/ is up to date. Thanks.